demo/icons/icon.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="../styles.css" type="text/css">
+		<script src="../../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import 'whatwg-fetch';
+			import 'demo-transfigurator/demo-transfigurator.js';
+			import '../../components/colors/colors.js';
+			import '../../components/typography/typography.js';
+			import '../../components/icons/icon.js';
+		</script>
+		<title>d2l-icon demo</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<script type="module">
+			import { addCustomStyle } from '../../custom-style-helper.js';
+			addCustomStyle(`
```

Comment:
It would be cool if they could use `css` here, i.e.

    addCustomStyle(css`...`);

Reply:
That might be possible.  The `css` template function produces a `CSSResult` such as:
![css-tempate-function](https://user-images.githubusercontent.com/9042472/56139863-b71e5780-5f67-11e9-87ff-9ea2bd2d1a78.png)

Lit obviously does some stuff with this that we'd need to do ourselves.  Probably something along [these lines](https://developers.google.com/web/updates/2019/02/constructable-stylesheets#using_constructed_stylesheets).  Not sure of non-supported browsers.

Reply:
One consequence to this is that they'd be required to import that `css` function.
```
import { css } from 'lit-element/lit-element.js';
```
Which would cause consumers to be directly dependent on `lit-element` code.

Reply:
Right. Why isn't there a perfect solution Dave? :P

Reply:
I feel like none of them are perfect, but all three are "pretty good".  I don't have big concerns with any of them.  Option 2 seems nice for disabling in the future, option 3 seems nice to avoid processing twice.

Reply:
Option 3 works if we need to load lazily (as does option 1), but Option 2 is time-sensitive.